### ISSUE TITLE
Amount field has no units

### ISSUE THREAD

**Issue by nroam** [thesgc/chembiohub_helpdesk#316]
Currently amount field has no units & no validation - you can type anything in. Including elephants.

**Comment by PaulBarrett79** [comment]
Hmm. Do we need an associated units dropdown for this field? Can custom fields have multiple subfields (for amount and units in other scenarios)?

**Comment by strets123** [comment]
The way this can work is that we add new field "types" for each of the units.

Each unit has an icon associated with it which we need to show on the front end

**Comment by KarenPorter** [comment]
High yellow.

**Comment by strets123** [comment]
Do as part of feature fields features

**Comment by KarenPorter** [comment]
@strets123 I don't think this is fully resolved,  please list outstanding requirements (and estimate if poss at this stage) and add to backlog if appropriate.

**Comment by strets123** [comment]
Need add a unit field when the user has selected a decimal or integer - simple pick from list. Also need to change the display of the edit and view data sections and do something in the Excel export, for example custom formatting

http://www.smallbusinesscomputing.com/buyersguide/article.php/3803431/How-to-Create-Custom-Formatting-in-Excel.htm

**Comment by strets123** [comment]
2 days work

**Comment by KarenPorter** [comment]
this is potentially important (though not sure why we haven't heard from existing users).
